apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/image-block-keywords/index.js

```diff
@@ -0,0 +1,24 @@
+/* global wp */
+
+/**
+ * External dependencies
+ */
+import { assign } from 'lodash';
+
+const additionalKeywords = [ 'logo', 'brand', 'emblem', 'hallmark' ];
+
+wp.hooks.addFilter(
```

Comment:
You could do this instead and let Webpack do the externalizing magic:

```
import { addFilter } from '@wordpress/hooks`;
```

That way `wp-hooks` gets added to dependencies automatically [at PHP side](https://github.com/Automattic/wp-calypso/blob/4f12e77962d146a870bceb1cb11a07ef0c758129/apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php#L167-L176).

Reply:
Thanks! I've updated the PR to do this. Would you mind double checking that it looks correct?

Reply:
Yup wp-hooks was added to https://382331-45936895-gh.circle-artifacts.com/0/tmp/artifacts/full-site-editing/full-site-editing-plugin/full-site-editing/dist/full-site-editing.deps.json